### ISSUE TITLE
Improve output format of `machinery config`

### ISSUE THREAD

**Issue by tgoettlicher** [SUSE/machinery#1271]
The output currently looks like this:
```
# machinery config
hints=false (Show hints about usage of Machinery in the context of the commands ran by the user)
remote-user=root (Defines the user which is used to access the inspected system via SSH)
experimental-features=true (Enable experimental features. See https://github.com/SUSE/machinery/wiki/Experimental-Features for more details)
perform-support-check=true (Check whether the current platform is supported by Machinery)
http-server-port=7585 (TCP port used by the HTTP server for the HTML view)
```

The equal signs should be equally aligned.

**Comment by cornelius** [comment]
The output should look like this:

```
# machinery config
hints                 = false (Show hints about usage of Machinery in the context of the commands ran by the user)
remote-user           = root  (Defines the user which is used to access the inspected system via SSH)
experimental-features = true  (Enable experimental features. See https://github.com/SUSE/machinery/wiki/Experimental-Features for more details)
perform-support-check = true  (Check whether the current platform is supported by Machinery)
http-server-port      = 7585  (TCP port used by the HTTP server for the HTML view)
```

**Comment by mschnitzer** [comment]
PR: https://github.com/SUSE/machinery/pull/1304

**Comment by keberle94** [comment]
Fixed by #1313
